.maestro/shared/onboarding_scripts/setup.js

```diff
@@ -0,0 +1,3 @@
+output.onboarding = {
+    runFullOnboarding: true
+}
```

Comment:
slightly awkward, but this is a mechanism to choose to skip onboarding if possible, and if the button doesn’t appear (unexpectedly) then the test can complete the full onboarding manually